app/assets/javascripts/jquery.wayfinding.js

```diff
@@ -292,9 +293,12 @@
 
 			// Make rooms clickable
 			$('#Rooms a', svgDiv).click(function (event) {
-				$(obj).trigger('wayfinding:roomClicked', [ { room_id : $(this).attr('id') } ] );
-				$(obj).wayfinding('routeTo', $(this).prop('id'));
-				event.preventDefault();
+				if(!animating) {
+					animating = true;
+					$(obj).trigger('wayfinding:roomClicked', [ { room_id : $(this).attr('id') } ] );
+					$(obj).wayfinding('routeTo', $(this).prop('id'));
+					event.preventDefault();
```

Comment:
We should still be firing off event.preventDefault() even if an animation is going on. It protects the browser from passing the event along to another method.